### ISSUE TITLE
Installation broken for Python > 3.8

### ISSUE THREAD

**Issue by WeilerP** [theislab/scvelo#1076]
The installation is broken when using Python > 3.8. The problem seems to be `scvi-tools==1.0.0`.

```
conda create -n dummy python=3.10 --yes && conda activate dummy
pip install -e ".[dev]" --no-cache
```

<details>

<summary> Error output </summary>

See [here](https://github.com/WeilerP/scvelo/actions/runs/5204095205).

</details>

**Comment by WeilerP** [comment]
@adamgayoso, any idea what might be causing this and how to fix it?

**Comment by WeilerP** [comment]
Seems to be fixed by #1079.
